lib/ci-stack.ts

```diff
@@ -33,19 +33,31 @@ export class CIStack extends Stack {
       },
     });
 
-    const useSslParameter = new CfnParameter(this, 'useSsl', {
-      description: 'If the jenkins instance should be access via SSL',
-      allowedValues: ['true', 'false'],
+    const useSslParameter = this.node.tryGetContext('useSsl');
+    if (useSslParameter !== 'true' && useSslParameter !== 'false') {
+      throw new ReferenceError('useSsl parameter is required to be set as "true" or "false"');
```

Comment:
Do we need to mention the quotes `"` as well? If not remove it. Just true or false seems fine

Reply:
I put quotes to emphasize on the 2 values that are acceptable.

Reply:
I think its understandable when you say it should be either true or false. The user should not get confused as to add the quotes and still see the error. Lets be clear :)